src/core/AnimatedReformat.js

```diff
@@ -0,0 +1,19 @@
+import AnimatedNode from './AnimatedNode';
+import { adapt } from '../core/AnimatedBlock';
+import { val } from '../val';
+
+class AnimatedReformat extends AnimatedNode {
```

Comment:
Now that I think about this node I'd prefer no leave it out of the PR. The only use case here is to transform color value and we need a web specific code path for instantiating it anyways. So maybe we can create a web specific color node that formats input values into `rgba(r,b,g,a)` string in `onEvaluate` instead?

Reply:
Alternatively we can consider using concat for web implementation there:
```
concat('rgba(', r, g, b, a, ')')
```

Reply:
I'm going to just delete color support and reformat since I don't like it either.